cmake/pml_cc_library.cmake

```diff
@@ -116,10 +116,12 @@ function(pml_cc_library)
     target_include_directories(${_NAME} SYSTEM
       PUBLIC
         "$<BUILD_INTERFACE:${PML_COMMON_INCLUDE_DIRS}>"
+        "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>"
```

Comment:
I don't think we want to generically do this, lots of headers aren't considered 'public' or 'external' to the project.